docs/install/remote.md

```diff
@@ -17,10 +17,38 @@ You can write your code on a local machine while running your functions on a rem
 Before you begin, ensure that the following prerequisites are met:
 
 Applications:
-- Python 3.9 
+- Python 3.9 (or Python 3.7 for Iguazio versions up to and including v3.5.2)
 - Recommended pip 22.x+
 
-## Set up client environment 
+The MLRun server is now based on Python 3.9. It's recommended to move the client to Python 3.9 as well. 
+
+MLRun v1.3.x maintains support for mlrun base images that are based on python 3.7. To differentiate between the images, the images based on
+python 3.7 have the suffix: `-py37`. The correct version is automatically chosen for the built-in MLRun images according to the Python version of the MLRun client (for example, a 3.7 Jupyter gets the `-py37` images).
```

Comment:
It's not accurate to say "3.7 Jupyter".
The correct termination will be a python 3.7 environment or iguazio <= 3.5.2 jupyter